src/components/tabs/__internal__/tabs-header/tabs-header.style.ts

```diff
@@ -25,6 +24,7 @@ const StyledTabsHeaderWrapper = styled.div<StyledTabsHeaderWrapperProps>`
       ${!isInSidebar &&
       css`
         min-width: 20%;
+        overflow-y: auto;
```

Comment:
**question**: once again there are multiple style changes in this file that don't seem to just affect the new focus styles

Reply:
Discussed and demoed with @harpalsingh, as the new approach uses box-shadow the overflow means some of the styling is hidden, these changes mean they are not. They were added for both so the component layout is consistent but we can conditionally apply them if Harpal thinks that's better